h2o-core/src/main/java/water/fvec/NewChunk.java

```diff
@@ -1537,7 +1542,7 @@ private Chunk chunkUUID() {
       if(idx >= 0)i = idx;
       else cancel_sparse(); // for now don't bother setting the sparse value
     }
-    _is[i] = _sslen;
+    if (_is != null) _is[i] = _sslen;
```

Comment:
Yeah Michal is right. The _is != null is only gonna hide problems. _is better not be null f it is it's better to throw NPE immediately.